auth.go

```diff
@@ -16,5 +16,5 @@ package errors
 
 // Unauthenticated returns an unauthenticated error
 func Unauthenticated(scheme string) Error {
-	return New(401, "unauthenticated for %s", scheme)
+	return New(401, "unauthenticated because of %s", scheme)
```

Comment:
Please also update the relevant unit tests, that rely on this message, as these are now broken because of this change.